### PR TITLE
Update dependency go modules for k8s v1.26.0-rc.0

### PR DESCRIPTION
**Pull request by sunnylovestiramisu**
Ran kubernetes-csi/csi-release-tools go-get-kubernetes.sh -p 1.26.0-rc.0

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,95 +7,77 @@ require (
...
+	k8s.io/kube-openapi v0.0.0-20221012153701-172d655c2280 // indirect
+	k8s.io/utils v0.0.0-20221107191617-1a15be271d1d // indirect
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
-	sigs.k8s.io/yaml v1.2.0 // indirect
+	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace k8s.io/api => k8s.io/api v0.25.0
+replace k8s.io/api => k8s.io/api v0.26.0-rc.0
```
Comment (xing-yang) [inline go.mod:70]: Can you remove the "replace"?

Comment (sunnylovestiramisu) [reply go.mod:70]: Do you mean remove all the replace or just this one replace k8s.io/api => k8s.io/api v0.26.0-rc.0?

File: go.mod [hunk 2]
```diff
@@ -7,118 +7,62 @@ require (
...
+	golang.org/x/term v0.1.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/grpc v1.51.0
-	k8s.io/api v0.25.0
-	k8s.io/apimachinery v0.25.0
-	k8s.io/apiserver v0.23.0
-	k8s.io/client-go v0.25.0
-	k8s.io/component-base v0.25.0
-	k8s.io/csi-translation-lib v0.23.0
+	google.golang.org/grpc v1.49.0
```
Comment (xing-yang) [inline go.mod:13]: This goes from v1.51.0 to v1.49.0, but I do see k/k has 1.49.0 as well.

**Pull request merged**